apps/docs/pages/guides/auth/auth-magic-link.mdx

```diff
@@ -46,6 +46,26 @@ async function signInWithEmail() {
 }
 ```
 
+</TabPanel>
+<TabPanel id="react-native" label="Expo React Native">
+Follow the [Expo Quickstart Guide](/docs/guides/getting-started/tutorials/with-expo-react-native#initialize-a-react-native-app) to corectly initialize `supabase-js` for usage with React Native.
+
+When your user signs in, call [signInWithOtp()](/docs/reference/javascript/auth-signinwithotp) with their email address:
```

Comment:
this is a bit confusing.

"when they sign in call signInWithOtp". So they have to sign in twice?